script/Arcueid/Day 4/04_00_ARC04_6_2C.txt

```diff
@@ -348,7 +377,8 @@ Mmm-hmm.
 [sha:eba13ac7b154d1a88fc240952e72c06a99d6036a]{
 -- Page 557, Offset 7070.
 -- こいつ、外国人……じゃなくて、外国吸血鬼のクセに日本文化に詳しいな。
-This foreigner... no, this foreign vampire knows quite a lot about Japanese culture for an outsider.
+// Arc weeb
+This foreigner... no, this foreign %{i}vampire%{/i} knows quite a lot about Japanese culture for an outsider.
```

Comment:
"for an outsider" appears to be an unnecessary creative liberty. I say cut the sentence off at "culture".

I'm guessing the idea was to make it stronger for the 「こいつ」, but "outsider" makes it sound like Shiki is angry at the fact that she's not Japanese rather than the fact that he's just irritated by her.